### ISSUE TITLE
[1.11.2][1.2.30] Mekanism Compat

### ISSUE THREAD

**Issue by Rosethorns** [BeetoGuy/BetterWithMods#461]
I'm not entirely sure if this should be with you Beeto or with Adian.
But the Hardcore Lumber setting has started interfering with the precision sawmill from Mekanism.

The HC Lumber setting causes the Mekanism precision sawmill to only produce sawdust and not the 6 planks it would normally otherwise produce.

I have no idea if recent changes to the HC Lumber setting are going to have knock on effects with other mods. But I imagine Adian isn't protecting his recipes which why I am not entirely sure this is your issue.

In short:
Better with Mods hardcore lumber setting is causing Mekanism precision sawmill lumber recipes to not work correctly.

**Comment by primetoxinz** [comment]
It is on our end, Mekanism is pulling right from the vanilla plank crafting recipes which we remove to replace it with ours. We already fix thermal expansion's sawmill by registering the recipe again, which is going to get bloated if we have to do this for every sawmill. I will look into finding a way to fix this without having to fix every other mod too.

**Comment by primetoxinz** [comment]
Fixed in #c4082e
